resources/src/effect/mod.rs

```diff
@@ -0,0 +1,562 @@
+use amount::Amount;
+use asset::{AssetIdentifier, Flag};
+use serde::{de, Deserialize, Deserializer};
+mod account_created;
```

Comment:
These modules make me think that it might be cleaner to just have a module per "type" of effect. This would mean `account`, `signer`, `trade`, `trustline`. Then have a handful of structs in each of those files.

Reply:
I think that you are correct, and I also think that we can do something similar with operations.  Given that this is such a huge chunk of code I would prefer to get this in and do those refactors in a separate PR.  What are your thoughts?

Reply:
Sure, we can do it as a separate PR. Can you make an issue for it for tracking?